apps/src/lib/tools/jsdebugger/JsDebugger.jsx

```diff
@@ -26,6 +26,7 @@ import {
 } from '../../../redux/watchedExpressions';
 import DebugConsole from './DebugConsole';
 import DebugButtons from './DebugButtons';
+// import experiments from '../../../util/experiments';
```

Comment:
Thanks for cleaning this up!

Reply:
Woops. I read that backwards. I thought you had originally deleted this line rather than added it. Either way, thanks for removing it :)